driving_log_replayer/scripts/perception_evaluator_node.py

```diff
@@ -340,7 +342,7 @@ def list_dynamic_object_from_ros_msg(
                 perception_object.classification
             )
             label = self.__evaluator.evaluator_config.label_converter.convert_label(
-                label=get_label(most_probable_classification)
+                name=get_label(most_probable_classification), attributes=[]
```

Comment:
[NITS] same with above
```suggestion
                name=get_label(most_probable_classification)
```